channel.go

```diff
@@ -9,10 +9,29 @@ import (
 // Channel represents an individual data channel being published by a Thing out
 // there in the world somewhere.
 type Channel struct {
-	ID           string          `json:"id"`
-	Metadata     []Metadata      `json:"metadata"`
-	Observations []Observation   `json:"observations"`
-	Type         schema.DataType `json:"type"`
+	ID           string        `json:"id"`
+	Metadata     []Metadata    `json:"metadata"`
+	Observations []Observation `json:"observations"`
+
+	// this is a bit confusing, can we use `json:"data_type"` instead?
+	Type schema.DataType `json:"type"`
+
+	// QuantityKind defines measurement type of quantity kind of this channel,
+	// in Ontology PoV, Channel is type QuantityKind, so type of channel can be anything that classified as quantitykind
+	// for example m3-lite:Temperature, m3-lite:WindSpeed
+	QuantityKind string `json:"isQuantityKind"`
```

Comment:
I don't have any strong feelings about it but I would consider keeping the JSON keys lowercase, perhaps something like `quantity_kind` rather than `isQuantityKind`?